test/remote/gateways/remote_card_stream_test.rb

```diff
@@ -113,6 +113,17 @@ def setup
       month: '12',
       year: '2020',
       brand: :visa)
+
+    @visacredit_three_ds_options = {
+      threeds_required: true,
+      three_d_secure: {
+        enrolled: 'true',
+        authentication_response_status: 'Y',
+        eci: '05',
+        cavv: 'Y2FyZGluYWxjb21tZXJjZWF1dGg',
+        ds_transaction_id: '00000000000004717472'
```

Comment:
💬  Comment:

Is this "ds_transaction_id" intentional ? or should it be "xid" as the unit tests

Reply:
fixed